test/integration/test_build.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+
+def test_build_fail_exitcode():
+    """Test that when a build fails, the ansible-builder exits with non-zero exit code.
+
+    Example: https://github.com/ansible/ansible-builder/issues/51
+    """
+    pytest.skip("Not implemented")
+
+
+def test_missing_python_requirements_file():
+    """If a user specifies a python requirements file, but we can't find it, fail sanely."""
+    pytest.skip("Not implemented")
+
+
+def test_missing_galaxy_requirements_file():
+    """If a user specifies a galaxy requirements file, but we can't find it, fail sanely."""
+    pytest.skip("Not implemented")
+
+
+def test_build_streams_output(cli, build_dir_and_ee_yml, ee_tag):
+    """Test that 'ansible-builder build' streams build output."""
+    tmpdir, eeyml = build_dir_and_ee_yml("")
+    result = cli(f"ansible-builder build -c {tmpdir} -f {eeyml} -t {ee_tag}")
+    assert f"podman build -f {tmpdir}/Containerfile" in result.stdout
```

Comment:
It would be a big help for several people to make it configurable podman/docker, because we have to spin up a VM or something to use podman. This will change as we phase out Mac laptops, but for the moment, it's been helpful to get going fast by swapping the two.

Reply:
This is going to leave around images isn't it? I think we need some additional layer in `conftest.py` on top of what you have to run a build command, so that it assigns a randomized tag, and makes sure to remove the tag in cleanup.

What you have is great, and I see that making sense as a separate work item.

Reply:
yes we can do something around that. Was going to probably make a fixture that detects available runtimes, and parametrizes this on that

Reply:
that sounds like more work than I had in mind.

Even if you just read from an (optional) environment variable, I think that would solve the problem.

Reply:
Using docker isn't really supported, but it's going to get developed against. That's kind of the split here in a nutshell.

Reply:
ack